### PR TITLE
Updated RX-78 machine / software information [Dave 'Foxhack' Silva]

### PR DESCRIPTION
**Pull request by FoxhackDN**

### CODE REVIEW

**Comment by FoxhackDN** [comment]
A note about the edit I made to the driver: The assumed 'incorrect inputs' for Challenge Golf were not wrong; the game uses the Home and End keys to start a 1P or 2P game, according to the overlay included with the game., so I changed that to 'possible'. Photos of these overlays will most likely be necessary to make sure that the inputs are correct.

File: hash/rx78.xml [hunk 1]
```diff
@@ -2,6 +2,82 @@
...
+
+The Bandai RX-78 Gundam has two slots where you can plug in a ROM cart or an additional device, like a
+printer interface. The system was capable of loading software from these ROM carts, or from a cassette
+tape through audio line-in ports to use with tape recorders; these ports were present in the BS Basic,
+Animation Graphics, Creative Graphics, Japanese Word Processor and Z80 Assembler carts. The system could
+use up to two carts or devices at the same time.
+
+There were a total of 20 games released for the system, plus educational and productivity software, and peripheral
+devices. One additional game and three expansion cards were announced but never released. The official translations
+printed in the game carts or tapes is used below where available. Names in brackets are unofficial translations.
```
Comment (cuavas) [inline hash/rx78.xml:14]: Grammar - should say “are used”

File: src/mame/drivers/rx78.cpp [hunk 2]
```diff
@@ -25,14 +25,40 @@ Driver by Angelo Salese & Robbbert.
 - To stop a cmt load, press STOP + SHIFT keys
 
 Known programs:
```
Comment (cuavas) [inline src/mame/drivers/rx78.cpp:27]: If the software list contains a list of all known software, it’s better to remove this list altogether.  Keeping it here just risks having it get out of sync.

File: hash/rx78.xml (hunk 1 above)
Comment (FoxhackDN) [reply hash/rx78.xml:14]: Er, wait, I didn't meant to tag this as resolve conversation. I'll fix the files, I just can't figure out how to, heh.

Comment (happppp) [reply hash/rx78.xml:14]: This PR was merged. Fixes can't be applied to it anymore, you'll just need to do a new PR.

**Pull request merged**